app/tests/test_main.py

```
@@ -79,3 +79,16 @@ def test_list_todos_api(self):
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response.json(), {"data": []})
         self.todo_service.list.assert_called_once_with(0, 10)
+
+    #너무 긴 설명문구 일때는 특정 문구를 반환한다.
+    def test_create_todo_api_with_too_long_description(self):
+        self.todo_service.create.side_effect = InvalidTodoDescriptionError()
```

Ruff (0.16.3)

[error] Undefined name `InvalidTodoDescriptionError`

(F821)